README.md

```diff
@@ -1,7 +1,7 @@
 # Azure Data Studio
 
 [![Join the chat at https://gitter.im/Microsoft/sqlopsstudio](https://badges.gitter.im/Microsoft/sqlopsstudio.svg)](https://gitter.im/Microsoft/sqlopsstudio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status](https://dev.azure.com/azuredatastudio/azuredatastudio/_apis/build/status/Azure%20Data%20Studio%20CI?branchName=main)](https://dev.azure.com/azuredatastudio/azuredatastudio/_build/latest?definitionId=4&branchName=main)
+[![Build Status](https://dev.azure.com/ms/azuredatastudio/_apis/build/status/AzureDataStudio-Localization-CI?branchName=main)](https://dev.azure.com/ms/azuredatastudio/_build/latest?definitionId=453&branchName=main)
```

Comment:
Why did you choose this pipeline? This is only the loc CI build so doesn't actually build the product or anything.

Reply:
Since the original pipeline was pointing to a CI build with the old pipeline name (Azure Data Studio CI), so I chose to update the link to point to the public org. We can modify it to product build too.

Reply:
We don't own that pipeline (and that's not really the point of that pipeline). 

Instead we should create a new pipeline in that repo and use that instead - with the https://github.com/microsoft/azuredatastudio/blob/main/.github/workflows/ci.yml definition.

Reply:
Created a DRI issue to follow up with this.